tests/demo/demo.cpp

```diff
@@ -21,12 +21,13 @@
 #include "scene_viknell.hpp"
 #include "scene_emibl.hpp"
 #include "scene_spheres.hpp"
+#include "scene_largescene.hpp"
 
 #include "model_loader_assimp.hpp"
 #include "model_loader_tinygltf.hpp"
 #include "d3d12/d3d12_dynamic_descriptor_heap.hpp"
 
-#define SCENE viknell_scene
+#define SCENE emibl_scene
```

Comment:
revert plzy

Reply:
revert, viktor is maaaad

Reply:
he jelous that emibl looks better than viknell.

Reply:
Seriously tho, revert. Emibl takes way too much to load